ada_feeding/ada_feeding/trees/acquire_food_tree.py

```diff
@@ -133,9 +173,171 @@ def create_tree(
                     inputs={"trajectory": BlackboardKey("trajectory")},
                     outputs={},
                 ),
+                # If Anything goes wrong, reset FT to safe levels
+                scoped_behavior(
+                    name="SafeFTPreempt",
+                    # pylint: disable=abstract-class-instantiated
+                    # It has trouble with py_trees meta classes
+                    pre_behavior=py_trees.behaviours.Success(),
+                    post_behavior=pre_moveto_config(
+                        name="PostAcquireFTSet", re_tare=False
+                    ),
+                    on_preempt_timeout=5.0,
+                    # Starts a new Sequence w/ Memory internally
+                    workers=[
+                        # Set Approach F/T Thresh
+                        retry_call_ros_service(
```

Comment:
I think it might be a bit clearer to pass this in as a pre-behavior instead, so that pre-behavior sets the FT thresh to potentially dangerous values and post-behavior fixes that. Functionally the two are the same, but I think the former would be more in the spirit of the scoped behavior.